src/shared/settingsholder.cpp

```diff
@@ -41,14 +40,14 @@ SettingsHolder* SettingsHolder::instance() {
   return s_instance;
 }
 
-SettingsHolder::SettingsHolder()
+SettingsHolder::SettingsHolder(const QString& appName)
```

Comment:
It seems like instead of needing to specify this explicitly, we could default to a sensible value derived from something like `qApp->applicationName()` or the name of the binary. That might cut down on the verbosity of specifying the settings name in the unit tests.